packages/core/core/src/Asset.js

```diff
@@ -47,7 +49,9 @@ type AssetOptions = {|
   outputHash?: string,
   env: Environment,
   meta?: Meta,
-  stats: Stats
+  stats: Stats,
+  symbols?: Map<Symbol, Symbol> | Array<[Symbol, Symbol]>,
```

Comment:
`symbols?: Iterable<[Symbol, Symbol]>` would cover both of these I think. I don't mind being explicit either.

Reply:
Nice